components/properties/src/maps.rs

```diff
@@ -265,6 +265,18 @@ impl<'a, T: TrieValue> CodePointMapDataBorrowed<'a, T> {
             .filter(move |r| r.value == val)
             .map(|r| r.range)
     }
+
+    /// Yields an [`Iterator`] returning ranges of consecutive code points that
+    /// do *not* have the value `v` in the [`CodePointMapData`].
+    pub fn iter_ranges_for_value_complemented(
```

Comment:
Question: Why are you exposing this specific case rather than full `iter_ranges_mapped`?

Reply:
I don't want to commit heavily to that API just yet.

The regex use case doesn't need custom mappers.